### PR TITLE
Fix conv2d bias

### PR DESCRIPTION
**Pull request by emailweixu**
1. The size of the bias parameter should be the number of filters. Changed LayerHelper.append_bias_op to allow this kind of bias.
2. Also change the default initialization of bias to be constant zero
3. Added a helper to get the value of a parameter.
4. Updated test_parameter

### CODE REVIEW

File: python/paddle/v2/fluid/layers.py [hunk 1]
```diff
@@ -676,6 +676,7 @@ def conv2d(input,
     filter_shape = [num_filters, num_filter_channels] + filter_size
 
     std = (2.0 / (filter_size[0]**2 * num_channels))**0.5
+    print 'name=', name, 'std=', std
```
Comment (jacquesqiao) [inline python/paddle/v2/fluid/layers.py]: this debug code should be removed

File: python/paddle/v2/fluid/io.py [hunk 2]
```diff
@@ -234,3 +234,34 @@ def load_inference_model(dirname, executor):
...
+    :param executor: executor for retrieving the value
+    :param name: the name of the parameter
+    :param program: the program where the variable is found
+    Default g_main_program.
+    :return: the LoDTensor for the variable
+    """
+    if program is None:
+        program = g_main_program
+    var = program.global_block().var(name)
+    assert is_parameter(var)
```
Comment (jacquesqiao) [inline python/paddle/v2/fluid/io.py]: should move this assert into get_parameter_value

File: python/paddle/v2/fluid/layers.py (hunk 1 above)
Comment (emailweixu) [reply python/paddle/v2/fluid/layers.py]: fixed.

File: python/paddle/v2/fluid/io.py (hunk 2 above)
Comment (emailweixu) [reply python/paddle/v2/fluid/io.py]: fixed

**Pull request merged**